src/utils/blueprints/blueprints-root.ts

```diff
@@ -6,5 +6,5 @@ const fileURL = import.meta.url;
 
 export const blueprintsRoot: string = join(
   getFilePath(fileURL),
-  '../../../../blueprints',
```

Comment:
This relative path, though it helps the tests pass, doesn't make sense because it points to a directory that doesn't exist.